File: 0392-is-subsequence/0392-is-subsequence.py
class Solution(object):
    def isSubsequence(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        i,count = 0,0
        while(i<len(t) and count<len(s)):
            if t[i]==s[count]:
                count +=1
            i +=1
        return count == len(s)

File: 0392-is-subsequence/0392-is-subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        char temp;
        int i = 0;
        int count = 0;
        while(i < t.length() && count<s.length()){
            if(s.charAt(count)==t.charAt(i)) count++;
            i++;
        }
        return count == s.length();
    }
}